next/components/forms/segments/UserProfile/UserProfileDetail.tsx

```diff
@@ -45,10 +47,28 @@ const UserProfileDetail = (props: UserProfileDetailProps) => {
 
   return (
     <div
-      className={cx(' flex flex-col', 'md:static md:z-0', {
+      className={cx('mt-3 flex flex-col bg-white', 'md:static md:z-0', {
```

Comment:
the mt-3 is creating this gray-ish block on top (see screenshot) - macOs chrome

<img width="1441" alt="Screenshot 2024-01-04 at 10 34 38" src="https://github.com/bratislava/konto.bratislava.sk/assets/2327312/e13cea9f-8acc-4eae-9b16-fa41a87f8261">

Reply:
Changed margin to padding